Client/Frontend/Browser/UserScriptManager.swift

```diff
@@ -240,6 +249,27 @@ class UserScriptManager {
         }
         return WKUserScript(source: source, injectionTime: .atDocumentStart, forMainFrameOnly: false)
     }()
+    
+    private let PlaylistSwizzlerScript: WKUserScript? = {
+        guard let path = Bundle.main.path(forResource: "PlaylistSwizzler", ofType: "js"), let source = try? String(contentsOfFile: path) else {
+            log.error("Failed to load PlaylistSwizzler.js")
+            return nil
+        }
+        return WKUserScript(source: source, injectionTime: .atDocumentStart, forMainFrameOnly: false)
+    }()
+    
+    private let PlaylistHelperScript: WKUserScript? = {
+        guard let path = Bundle.main.path(forResource: "Playlist", ofType: "js"), let source = try? String(contentsOfFile: path) else {
+            log.error("Failed to load Playlist.js")
+            return nil
+        }
+        
+        var alteredSource = source
+        let token = UserScriptManager.securityToken.uuidString.replacingOccurrences(of: "-", with: "", options: .literal)
+        alteredSource = alteredSource.replacingOccurrences(of: "$<videosSupportFullscreen>", with: "PVSF\(token)", options: .literal)
```

Comment:
`$<videosSupportFullscreen>` is not used in `Playlist.js`

Reply:
Fixed.